src/NServiceBus.Testing/NServiceBus.Testing.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="Fody" Version="6.4.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.1" />
```

Comment:
With https://github.com/Particular/NServiceBus/pull/5970 merged, this package reference can be removed.